app/router.go

```diff
@@ -188,3 +195,31 @@ type optionValue struct {
 }
 
 func nop(r render.Renderer) render.Renderer { return r }
+
+func addKubernetesTopologies() {
```

Comment:
The thought was to have it show the button if there had *ever* been k8s data received, so that if there was no k8s data in one report the button wouldn't hide again. But, I suppose we could handle that in the UI. Adding topologies makes sense from a plugin sense, but for something baked in it *does* seem a bit haphazard.